pocketbase/umbrel-app.yml

```diff
@@ -0,0 +1,41 @@
+manifestVersion: 1
+id: pocketbase
+name: PocketBase
+tagline: Open Source backend for your next SaaS and Mobile app in 1 file
+category: developer
+version: "0.22.11"
+port: 5400
+description: >-
+  PocketBase is an open source backend consisting of embedded database (SQLite) with realtime subscriptions, built-in auth management, convenient dashboard UI and simple REST-ish API.
+
+
+  ⏱️ Realtime database
+  
+  Embedded performant database with schema builder, data validations, realtime subscriptions and easy to use REST api.
+
+
+  🪪 Authentication
+  
+  Manage your app users and handle email/password and OAuth2 sign ups (Google, Facebook, GitHub, GitLab) without the hassle.
+
+  
+  💾 File storage
+  
+  Sanely store files locally or in a S3 storage. Easily attach media to your database records and generate thumbs on the fly.
+  
+  
+  🧩 Extendable
+  
+  Use as a standalone app OR as a framework, that you can extend via Go and JavaScript hooks to create your own custom portable backend.
+developer: Gani
```

Comment:
Let's use `PocketBase` here as the developer. The github repo is under the PocketBase organization and the website shows this:

<img width="188" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/46d9d465-c15d-4367-a1fd-8a6957bcd625">